src/grammar.rs

```diff
@@ -43,7 +43,7 @@ mod test {
   #[test]
   fn print_grammar_string() {
     let g :String = get_grammar_string();
-    info!("alles: {:?}", g);
+    println!("alles: {:?}", g);
```

Comment:
Are you initializing a logger in the test code? When using env_logger, I think you need to initialize it in every test run, as they're run in parallel.